### PR TITLE
Add support for parsing the config values from the Triton protobuf

### PR DESCRIPTION
**Pull request by Tabrizian**
~~This needs to be merged after #77 and #69.~~

### CODE REVIEW

**Comment by Tabrizian** [comment]
You can click on the individual commits to see the related changes.

File: model_analyzer/config/config_value.py [hunk 1]
```diff
@@ -85,23 +83,33 @@ def value(self):
...
+            for key, value_ in return_result.items():
                 if hasattr(value_, 'value'):
-                    return_result[key] = value_.value()
+                    final_return_result[key] = value_.value()
                 else:
-                    return_result[key] = value_.value()
-        elif type(self._value) is list:
-            return_result = []
-            for item in self._value:
+                    final_return_result[key] = value_.value()
```
Comment (xprotobeast2) [inline model_analyzer/config/config_value.py:95]: Should this be 

```
final_return_result[key] = value_
```

Seems like both branches are the same

File: model_analyzer/triton/model/model_config.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            )
+
+        if os.path.isfile(model_path):
+            raise TritonModelAnalyzerException(
+                'Model output path must be a directory.')
+
+        with open(os.path.join(model_path, "config.pbtxt"), 'r+') as f:
+            config_str = f.read()
+
+        print(config_str)
```
Comment (xprotobeast2) [inline model_analyzer/triton/model/model_config.py:63]: Do we need this `print`

**Review by xprotobeast2 — CHANGES_REQUESTED** [review]

File: model_analyzer/config/config_value.py (hunk 1 above)
Comment (Tabrizian) [reply model_analyzer/config/config_value.py]: Good catch! The other branch should never happen. I replaced it with an exception.

File: model_analyzer/triton/model/model_config.py (hunk 2 above)
Comment (Tabrizian) [reply model_analyzer/triton/model/model_config.py]: Right, I removed it.

**Review by xprotobeast2 — APPROVED** [review]

**Pull request merged**